src/uinput.rs

```diff
@@ -27,7 +26,7 @@ fn copy_name(dest: &mut [c_char; UINPUT_MAX_NAME_SIZE as usize], name: &[u8]) ->
         unsafe {
             ptr::copy_nonoverlapping(name.as_ptr() as *const _, dest.as_mut_ptr() as *mut _, name.len());
         }
-        dest[name.len()] = 0 as _;
+        dest[name.len()] = 0_i8;
```

Comment:
I believe this needs to remain `as c_char`, because it is not necessarily signed on all architectures (or remove the `as` entirely and let inference do it?)

Reply:
whoops. Yep, that'll cause a compile error on platforms with unsigned char (arm). I also ran `cargo check --target=aarch64-unknown-linux-gnu` to make sure I didn't make another similar error.